lib/rubocop/cop/rails/content_tag.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks that `tag` is used instead of `content_tag`
+      # because `content_tag` is legacy syntax.
+      #
+      # @example
+      #  # bad
+      #  content_tag(:p, 'Hello world!')
+      #  content_tag(:br)
+      #
+      #  # good
+      #  tag.p('Hello world!')
+      #  tag.br
+      class ContentTag < Cop
+        include RangeHelp
+        extend TargetRailsVersion
+
+        minimum_target_rails_version 5.1
+
+        MSG = 'Use `tag` instead of `content_tag`.'
+
+        def on_send(node)
+          return unless node.method?(:content_tag)
+
+          add_offense(node)
+        end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            if method_name?(node.arguments.first)
+              replace_method(corrector, node)
+              remove_first_argument(corrector, node)
+            else
+              corrector.replace(node.loc.selector, 'tag')
+            end
+          end
+        end
+
+        private
+
+        def method_name?(node)
+          return false unless node.str_type? || node.sym_type?
+
+          /^[a-zA-Z_][a-zA-Z_0-9]*$/.match?(node.value)
+        end
+
+        def replace_method(corrector, node)
```

Comment:
It may be able to add a little more information :-)

```suggestion
        def replace_with_tag_method(corrector, node)
```